scripts/docker/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
you might want to set a version, to ensure that we will not be affected by changes in request python library

Reply:
set to 2.32.3